File: CriminalIntent2/app/src/main/java/com/example/criminalintent/CrimeActivity.java
package com.example.criminalintent;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentActivity;
import androidx.fragment.app.FragmentManager;

import android.os.Bundle;

public class CrimeActivity extends FragmentActivity {


    FragmentManager fragmentManager;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        fragmentManager = getSupportFragmentManager();
        fragmentManager.beginTransaction().add(R.id.fragment_container,new CrimeFragment()).commit();

    }
}

File: app/src/main/java/com/example/criminalintent/CrimeActivity.kt
package com.example.criminalintent

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.fragment.app.FragmentManager

class CrimeActivity : AppCompatActivity() {

    lateinit var fragManager :FragmentManager
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        fragManager = supportFragmentManager
        fragManager.beginTransaction().add(R.id.fragment_container,CrimeDetailFragment()).commit()


    }
}